TanukiKitTests/ProjectTests.swift

```diff
@@ -35,11 +35,11 @@ class ProjectTests: XCTestCase {
 
     func testFailToGetProjects() {
         let json = "{\"message\":\"401 Unauthorized\"}"
-        let session = TanukiKitURLTestSession(expectedURL: "https://gitlab.com/api/v3/projects?page=1&per_page=100", expectedHTTPMethod: "GET", response: json, statusCode: 401)
```

Comment:
`&search=&simple=&sort=desc&visibility=`

This looks very weird. If the parameter name is not given it should not be added to the request. Empty parameter names can leas to unexpected behavior.